### PR TITLE
Balance Info Repository

### PR DESCRIPTION
**Pull request by Horat1us**

### CODE REVIEW

File: src/Info/Response.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    {
+        $result = "";
+        switch ($this->type) {
+            case '1':
+                $result .= "************PRIVAT************\n";
+                break;
+            default:
+                $result .= "************DIAMANT************\n";
+        }
+        $result .= "Ключ = {$this->id}\n";
```
Comment (sashabeton) [inline src/Info/Response.php:132]: Maybe better PHP_EOL?

**Review by sashabeton — DISMISSED** [review]

File: src/Info/Response.php (hunk 1 above)
Comment (Horat1us) [reply src/Info/Response.php:132]: No matter. Copied from Platon example

File: src/Info/InvalidResponseException.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Wearesho\Bobra\Platon\Info;
+
+use Throwable;
```
Comment (KartaviK) [inline src/Info/InvalidResponseException.php]: Excess include class from global namespace

File: src/Info/Repository.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    /**
+     * @param \DateTimeInterface $date
+     * @return Response[]
+     * @throws GuzzleHttp\Exception\GuzzleException
+     * @throws InvalidResponseException
+     */
+    public function get(\DateTimeInterface $date = null): array
+    {
+        $response = $this->client->request("POST", 'http://62.113.194.121/credit/api/v1/check', [
```
Comment (KartaviK) [inline src/Info/Repository.php]: Wouldn't it be better to put the url in the config?

**Pull request merged**